internal/queries/rate.go

```diff
@@ -47,9 +47,10 @@ func createRateTradeQuery(source, dest Asset, startUnixTimestamp, endUnixTimesta
 	counterAssetMatch := fmt.Sprintf("((C.asset_code=\"%s\" AND C.asset_issuer=\"%s\") OR (B.asset_code=\"%s\" AND B.asset_issuer=\"%s\"))",
 		source.Code, source.Issuer, dest.Code, dest.Issuer)
 	counterAssetSelect := "SUM(T.base_amount)/SUM(T.counter_amount)"
+	titleField := getTitleField("L.sequence", "L.closed_at", aggregateBy)
```

Comment:
Right before assembling the query, could you comment what the expected query would look like? This would be helpful in a few different places, as these queries are becoming more complex.

Reply:
(Basically, just add a sample query just above where you start assembling said query, so an external user can come in, read it, and understand what the query should look like.)